app/lib/notifications.server.ts

```diff
@@ -4,27 +4,28 @@ import logger from './logger.server';
 
 import type { SendMailOptions } from 'nodemailer';
 
-const { NOTIFICATIONS_EMAIL_USER, NODE_ENV, MAILHOG_SMTP_PORT } = process.env;
-const { NODEMAILER_CLIENT_ID, NODEMAILER_ACCESS_TOKEN } = secrets ?? {};
+const { NOTIFICATIONS_EMAIL_USER, NODE_ENV, MAILHOG_SMTP_PORT, NOTIFICATIONS_SMTP_PORT } =
+  process.env;
+const { NOTIFICATIONS_USERNAME, NOTIFICATIONS_PASSWORD } = secrets ?? {};
 
 const initializeTransport = () => {
   if (!NOTIFICATIONS_EMAIL_USER) {
     throw new Error('Missing Nodemailer user. Skipping nodemailer configuration');
   }
   if (NODE_ENV === 'production') {
-    if (!NODEMAILER_CLIENT_ID || !NODEMAILER_ACCESS_TOKEN) {
+    if (!NOTIFICATIONS_USERNAME || !NOTIFICATIONS_PASSWORD) {
       throw new Error(
         'Missing Nodemailer environment variables or docker secrets. Skipping nodemailer configuration'
       );
     }
     return createTransport({
-      service: 'Outlook365',
+      host: 'smtp.office365.com',
+      port: Number(NOTIFICATIONS_SMTP_PORT || '587'),
```

Comment:
This will always be `587` in production, let's not bother with a variable to configure it.